python/pyarrow/tests/conftest.py

```diff
@@ -199,11 +200,11 @@ def pytest_configure(config):
             "markers", mark,
         )
 
-        flag = '--{}'.format(mark)
         enable_flag = '--enable-{}'.format(mark)
+        disable_flag = '--disable-{}'.format(mark)
 
-        is_enabled = (config.getoption(flag) or
-                      config.getoption(enable_flag))
+        is_enabled = (config.getoption(enable_flag) and not
+                      config.getoption(disable_flag))
```

Comment:
Why the complication? `--xxx` is much nicer than `--enable-xxx`. And `--enable-xxx --disable-xxx` doesn't make sense.

Reply:
The absence of `--xxx` means to use the default for that flag which depends on an optional import most of the cases.

Reply:
I see, thanks.